src/Patterns/GetInformed.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * Get Informed pattern class.
+ *
+ * @package P4\MasterTheme\Patterns
+ * @since 0.1
+ */
+
+namespace P4\MasterTheme\Patterns;
+
+/**
+ * Class Get Informed.
+ *
+ * @package P4\MasterTheme\Patterns
+ */
+class GetInformed extends BlockPattern
+{
+    /**
+     * @inheritDoc
```

Comment:
Should we add a proper function description?

Reply:
Same here!